rpcs3/rpcs3qt/rpcn_settings_dialog.cpp

```diff
@@ -37,6 +37,7 @@ rpcn_settings_dialog::rpcn_settings_dialog(QWidget* parent)
 	QPushButton* btn_chg_pass = new QPushButton(tr("Set Password"));
 	QLabel *label_token       = new QLabel(tr("Token:"));
 	m_edit_token              = new QLineEdit();
+	m_edit_token->setMaxLength(16);
```

Comment:
Is this magic number not available as a constant somewhere in rpcn code?

Reply:
No, it's something completely unrelated to ps3, it's token to verify email...